lib/src/mobile_scanner_platform_interface.dart

```diff
@@ -83,6 +83,14 @@ abstract class MobileScannerPlatform extends PlatformInterface {
     throw UnimplementedError('setZoomScale() has not been implemented.');
   }
 
+  /// Set the focus position for the camera.
```

Comment:
A suggestion for the doc comment:

```suggestion
  /// Set the focus position for the camera.
  ///
  /// The provided point should be in the range `(0,0) - (1,1)`, where `(0,0)` is the top left
  /// and `(1,1)` is the bottom right.
```